### PR TITLE
WFPrev-133 Graalifying the API

### PR DESCRIPTION
**Pull request by agordon-vivid**

### CODE REVIEW

**Review by dhlevi — APPROVED** [review]
Obviously we worked together on a bunch of this so I'd like others to review things first.

We may want to bring the spring doc stuff back in now and see if it works OK, but it might need some massaging to find the bits that need to go into the reflection config.

File: server/wfprev-api/src/main/resources/native-image/reflection-config.json [hunk 1]
```diff
@@ -0,0 +1,505 @@
...
+  {
+    "name": "ca.bc.gov.nrs.wfone.common.webade.oauth2.token.client.resource.CheckedToken",
+    "allDeclaredConstructors": true,
+    "allPublicConstructors": true,
+    "allDeclaredMethods": true,
+    "allPublicMethods": true,
+    "allDeclaredFields": true,
+    "allPublicFields": true
+  }
+]
```
Comment (ssylver93) [inline server/wfprev-api/src/main/resources/native-image/reflection-config.json:505]: Does every newly imported class need to get configured here in reflection-config.json/what is the criteria to add a class to the configuration here?

**Review by ssylver93 — APPROVED** [review]

**Comment by yzlucas** [comment]
I spot some potential redundancy in reflection-conflig.json with same calss name but slightly differing configurations. Nothing major but we may want to clean them up?

File: server/wfprev-api/src/main/resources/native-image/reflection-config.json (hunk 1 above)
Comment (agordon-vivid) [reply server/wfprev-api/src/main/resources/native-image/reflection-config.json:505]: Hopefully not

**Pull request merged**